app/elements/coder/profile/user-challenges-table.html

```diff
@@ -0,0 +1,92 @@
+<link rel="import" href="/bower_components/vaadin-grid/vaadin-grid.html">
+<link rel="import" href="/bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="/bower_components/iron-icons/social-icons.html">
+<link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="/bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="/bower_components/iron-ajax/iron-ajax.html">
+
+<link rel="import" href="/elements/common/data-providers/challenge/user-challenges-provider.html">
+
+<dom-module id="user-challenges-table">
+	<style include="iron-flex">
+		:host {
+			display: block;
+			height: 100%;
+		}
+		paper-icon-button {
+			@apply(--layout-self-center);
+		}
+	</style>
+	<template>
+		<user-challenges-provider data="{{challenges}}"></user-challenges-provider>
+		<vaadin-grid id="challengesGrid" class="fullHeight" items="[[challenges]]" on-selected-items-changed="_selectedItemsChanged">
+			<table>
+				<thead>
+					<tr>
+						<th>Name</th>
+						<th>Start date</th>
+						<th>End date</th>
+						<th>Status</th>
+					</tr>
+				</thead>
+				<colgroup>
+					<col name="challenge.name"/>
+					<col name="challenge.startDate"/>
+					<col name="challenge.endDate"/>
+					<col name="status"/>
+				</colgroup>
+			</table>
+		</vaadin-grid>
+	</template>
+	<script type="application/javascript" src="/scripts/behaviors/providers/user-provider-behavior.js"></script>
+	<script>
+		// NOTE: members needs to be set for data to be shown.
+		Polymer({
+			is: 'user-challenges-table',
+			behaviors: [
+				Behaviors.Providers.User
+			],
+			observers: [
+				'_dataChanged(challenges)'
+			],
+			_selectedItemsChanged: function(){
+				var selected = this.$.challengesGrid.selection.selected();
+				if(!selected || selected.length === 0){
+					return;
+				}
+				var item = this.challenges[selected[0]];
+				if(item.status === 'COMPLETED'){
```

Comment:
IMO I don't think it's nice UX to just redirect when a row is clicked in the table. Mby people randomly want to highlight something in a table.